core/controllers/admin.py

```diff
@@ -1551,6 +1552,24 @@ def put(self) -> None:
         user_services.set_username(user_id, new_username)
         user_services.log_username_change(
             self.user_id, old_username, new_username)
+
+        # Update profile picture.
+        old_fs = fs_services.GcsFileSystem(
+            feconf.ENTITY_TYPE_USER, old_username)
+        new_fs = fs_services.GcsFileSystem(
+            feconf.ENTITY_TYPE_USER, new_username)
+        if old_fs.isfile('profile_picture.png'):
+            image_png = old_fs.get('profile_picture.png')
+            old_fs.delete('profile_picture.png')
+            new_fs.commit(
+                'profile_picture.png', image_png, mimetype='image/png')
+
+        if old_fs.isfile('profile_picture.webp'):
+            image_webp = old_fs.get('profile_picture.webp')
+            old_fs.delete('profile_picture.webp')
+            new_fs.commit(
+                'profile_picture.webp', image_webp, mimetype='image/webp')
```

Comment:
I think we should error in the case when the image does not exist.

Reply:
Updated!